Fixed some inaccuracies and typos. (#97)
Joystream/helpdesk: roles/content-curators/README.md

- If you attempt to apply with insufficient stake.
- If you try to stake more tokens than you can (note the application transaction costs 1 JOY)
- If you apply for a role with a maximum number of slots, with fixed or no stake requirements, and the maximum application slots have been reached.

> - If you try to apply manually through the `Extrinsic` sidebar, you can fail for a variety of other reasons.

Reviewer: ```suggestion
- If you try to apply manually through the `Extrinsics` sidebar, you can fail for a variety of other reasons.
```
Seems OK in the original form.

### Regular Checks
At some defined (minimum) frequency, the Curator reviews all new videos and channels uploaded.

> To avoid everyone stepping on each other’s toes, the Lead should consider splitting the workload between the team, e.g., by day/time and if required, also by language, category, etc.

Reviewer: ```suggestion
To avoid everyone stepping on each other’s toes, the Lead should consider splitting the workload between the team, e.g. by date/time and if required, also by language, category, etc.
```

The Curator team should also monitor this thread, as it would be where the channel owner reports back to them if they made change(s) requested, disagrees with some action taken, or simply has any questions.

### Council Reports

> At some agreed interval, assumed to be at least once for each Council Term, _all_ videos and channels must be "checked in" on. There could be many reasons why  to verify that they are still acceptable. In some cases, some details may have been overlooked in previous "checkups", but there is also the chance that a Channel owner or a (rogue) Curator makes a change that requires an action.

Reviewer: ```suggestion
At some agreed interval, assumed to be at least once for each Council Term, _all_ videos and channels must be "checked in" on. There could be many reasons why  to verify that they are still acceptable. In some cases, some details may have been overlooked in previous "check-ups", but there is also the chance that a Channel owner or a (rogue) Curator makes a change that requires an action.
```

**When to make a video `hidden`:**
- The content must be as "advertised"
  - If the title, description, category and thumbnail imply a baking video, the video should not be a documentary about Bitcoin
- Missing artwork

>   - If thumbnail is missing, or just really poor, the video can be hidden, or given a warning (Council decides u.n.o.)

Reviewer: ```suggestion
  - If thumbnail is missing, or just really poor, the video can be hidden, or given a warning.
```

- Poor quality
  - If the video quality is "unreasonably" low

**When to `censor` a video:**
- License requires attribution
  - If the selected license requires attribution, but none is given the, or attribution is incorrect
- Breach of license

>   - If the license has specific requirements with it (e.g., "no derivates" if [CC_BY_NC_ND](https://creativecommons.org/licenses/by-nc-nd/4.0/))

Reviewer: ```suggestion
  - If the license has specific requirements with it (e.g. "no derivatives" if [CC_BY_NC_ND](https://creativecommons.org/licenses/by-nc-nd/4.0/))
```
